docs/configuration.md

```diff
@@ -2939,6 +2939,10 @@ To specify a different configuration directory other than the default "SPARK_HOM
 you can set SPARK_CONF_DIR. Spark will use the configuration files (spark-defaults.conf, spark-env.sh, log4j.properties, etc)
 from this directory.
 
+# Overriding classpath default directory
+
+To specify a different classpath directory other than the default "SPARK_HOME/jars"  you can set SPARK_JARS_DIR. Any of the services under "SPARK_HOME/sbin" folder will use it.
```

Comment:
nit, `"  you` -> `",  you`

Reply:
Why do we limit to only `sbin` here?

Reply:
Like the PR title have `bin` and `sbin`, this PR changes the standalone ExternalShuffleService like the following. We had better mention that.
```
bin/spark-class org.apache.spark.deploy.ExternalShuffleService
```